tests/jenkins/_env_setup.sh

```diff
@@ -10,7 +10,10 @@ export SPACK_ROOT="${BUILD_HOME}/spack"
 export SPACK_INSTALL_PREFIX="${SPACK_INSTALL_PREFIX:-${INSTALL_HOME}}"
 export SOFTS_DIR_PATH=$SPACK_INSTALL_PREFIX  # Deprecated, but might still be reqd
 export PATH=$SPACK_ROOT/bin:/usr/bin:$PATH
```

Comment:
Also this I believe that it's not needed

Reply:
removed